NYTPhotoViewer/NYTPhotosViewController.h

```diff
@@ -177,6 +182,21 @@ extern NSString * const NYTPhotosViewControllerDidDismissNotification;
 
 @optional
 
+/**
+ *  Called when the view is about to made visible.
+ *
+ *  @param photosViewController The `NYTPhotosViewController` instance that sent the delegate message.
```

Comment:
Backticks don't work for Obj-C doc comments. You want `@p` for "preformatted" or `@c` for "code". Go with whichever one the code base already uses, if any. Same for the next property.

Reply:
Really?  This and other headers in the project use backticks in dozens of places.  NSHipster's [article on appledoc](https://nshipster.com/objective-c-documentation/) gives 4 examples to "well-documented Objective-C projects", all four use backticks this way.

Reply:
Try one in Obj-C, then Option-click on the property (or look in the Help Inspector) and see how the formatting is. Maybe they added it more recently? Or maybe it always worked, but I'm pretty sure it didn't.

Reply:
I see what you mean now.

<img width="279" alt="image" src="https://user-images.githubusercontent.com/325572/101985023-4c00b900-3c53-11eb-86d7-5b417152d0bd.png">

Reply:
if we make this change (all backticks to @c), let's do it in a different PR, to avoid cluttering the "real" diffs of this PR.

Reply:
i create #339 to track the 'change backtick' task. marking this convo 'resolvde'.